templates/citizen_project/edit.html.twig

```diff
@@ -26,7 +26,13 @@
                 {{ include('components/caret--left.html.twig') }}
                 Retour au projet citoyen</a>
 
-            {% if not citizen_project.isApproved %}
+            {% if citizen_project.isRefused %}
+                <div class="committee__waiting-for-approval">
+                    En l'état, votre projet citoyen n'a pas été validé par nos équipes.
+                    Vous pouvez à tout moment modifier ce projet ou en proposer un nouveau (dans la limite de 3 projets par personne).
+                    Pour plus d’informations, reportez-vous à la Charte des Projets Citoyens <a class="link--blue" href="https://storage.googleapis.com/en-marche-prod/documents/projets-citoyens/La%20Charte%20des%20Projets%20Citoyens.pdf">la Charte</a>.
```

Comment:
are you sure for a position of the link? it's `à La Charte des Projects Citoyens La Charte.`